functional-tests/README.md

```diff
@@ -51,8 +53,35 @@ Only on windows:
 
 Only on MacOS:
 
-    ./gradlew safariTest -DsafariTest.single=CustomJUnitSpecRunner
+    DEPRECATED ./gradlew safariTest -DsafariTest.single=CustomJUnitSpecRunner
+
+## Run tests in project root
+
+The tests can also be run from the projects root using the command 'make test'
+
+## Modifying tests that run in the pipeline
+
+The shell script controlling how the tests in the pipeline are run is run_test.sh. If new products need to be added to the
+GEB testing process this is where they are added.
 
 ## Test Report
 
 Report can be found under `/functional-tests/build/reports/tests`
+
+## Common Problems
+
+- Data clean up FAILS:
+
+This often occurs when a new table is added to the database and the clean up script attempts to delete a linked table. Often
+running the test on a clean database will work. Though sometimes the data_deletion.sql script will have to be modified to
+delete dependancies before proceeding.
+
+- The login test fails locally:
+
+If you've never managed to run the functional tests locally before, make certain your .env file matches the .env example.
+You will need to get the password from openshift secrets.
+
+- data clean up fails with an error like `org.postgresql.util.PSQLException: FATAL: role "mds" does not exist`:
+
+You may be running postgress locally for a different project. Make certain to stop that process, then run `make database`
```

Comment:
*postgres 
*root

Also the error means, that there is no user called mds. That has nothing to do with running postgres locally or not. Maybe the description could be, ensure you have the correct database running with the user and schema created or something like that?

Reply:
done